# Copyright 2023 Nocompila
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

#
# robot usado: tiago - kobuki
#   tiago:
#     input_scan: /scan_raw
#     output_vel: /nav_vel
#   kobuki:  
#     input_scan: /scan
#     output_vel: /cmd_vel

nc_bt_patrolling:
  robot: tiago
  input_scan: /scan_raw
  output_vel: /nav_vel
  type: simulaton
  way_points:
    wp_1: [1.13, 3.07]  # gym
    wp_2: [-5.22, -0.66]  # bed
    wp_3: [-4.26, -3.56]  # pilatex
    wp_4: [1.10, -4.38]  # sofa

use_sim_time: True